### DISCUSSION TITLE
Not Appearing

### DISCUSSION THREAD

**Discussion by iomari** [starship/starship#5053] [Q&A] [answered]
Greetings,
I currently have a full .zshrc config that I've been using for years with powerlevel10k as my prompt.
To test starship,I added eval "$(starship init zsh)" as my very last line in my .zshrc file. I was expecting to see the default Starship prompt but I'm still seeing my powerlevel prompt.

Why is this?

iomari

**Comment by davidkna** [comment, marked as answer]
Starship only sets up its handling of the `PROMPT` once, which mean that other prompts that are still enabled may overwrite the starship prompt. Please ensure that any other prompts do not get enabled before enabling starship, or try running `prompt off` first.

**Comment by iomari** [comment]
Thanks. Problem resolved
